docs/docs/writing-stories.mdx

```diff
@@ -5,11 +5,427 @@ title: Writing Stories
+  	- reviewed_transactions: ["target", "starbucks"]
+  - intent: affirm
+  - action: utter_confirm_transaction_dispute
+  - action: utter_replace_card
+  - action: mailing_address_form
+  - active_loop: mailing_address
+  - active_loop: null
+  - action: utter_sent_replacement
+  - action: utter_anything_else
+  - intent: affirm
+  - action: utter_help
+```
+
+Handling a lost credit card involves a series of sub-tasks, namely checking spending history for fraudulent transactions, confirming a mailing address for a replacement card, and then following up with the user with any additional requests. In this conversation arc, there are several places where the bot prompts for user input, creating branching paths that need to be accounted for. For example, when prompted with "utter_ask_fraudulent_transactions", the user might respond with a "deny" intent if none are applicable. The user might also choose to respond with a "deny" intent when asked if there's anything else the bot can help them with.
+
+We can separate out this long story into several smaller stories as:
+
+```yaml-rasa
+- story: Customer loses a credit card
+  steps:
+  - intent: card_lost
+  - action: utter_card_locked
+  - action: spending_history_form
+  - active_loop: spending_history_form
+  - active_loop: null
+  - slot_was_set:
+  	- reviewed_transactions: ["starbucks"]
+  - action: utter_ask_fraudulent_transactions
+
+- story: Customer reviews transactions and gets a new card
+  steps:
+  - action: utter_ask_fraudulent_transactions
+  - intent: inform
+  - action: action_update_transactions
+  - slot_was_set:
+  	- reviewed_transactions: ["target", "starbucks"]
+  - intent: affirm
+  - action: utter_confirm_transaction_dispute
+  - action: utter_replace_card
+  - action: mailing_address_form
+  - active_loop: mailing_address
+  - active_loop: null
+  - action: utter_sent_replacement
+  - action: utter_anything_else
+
+## Customer has something else they need help with
+- story: Customer has something else they need help with
+  steps:
+  - action: utter_anything_else
+  - intent: affirm
+  - action: utter_help
+```
+
+### Managing Story Files
+
+You can provide training data to Rasa Open Source in Markdown format as a single file or as a directory containing multiple files. When writing stories, it's usually a good idea to create separate story files based on the types of conversations being represented. For example, you might create a file `chitchat.md` for handling chitchat stories and an `faqs.md` file for FAQ's. Refer to our rasa-demo bot (link here) for examples of story file management in complex assistants.
+
+
+
+## Context Switching
+
+Often, users will not respond with the information you ask of them and instead deviate
+from the happy path with unrelated questions. Using [CDD](./conversation-driven-development.mdx) to understand
+what unhappy paths your users are taking, you can create stories for handling context
+switching.
+
+
+### When to use Rules
```

Comment:
This header should match the other, 
```suggestion
### Using Rules for Context Switching
```
and `Using Stories for Context Switching` (i say use instead of write because in the case of rules, you're not writing them as you've pointed out)